### ISSUE TITLE
ANR (locked <0x034e0bee> (a com.google.android.gms.ads.nonagon.ad.activeview.e)

### ISSUE THREAD

**Issue by shewenbiao** [googleads/googleads-mobile-android-examples#167]
SDK Version:  16.0.1 or 17.2.0
 
ANR log (Biggest ANR cause of my app):
```java
"main" prio=5 tid=1 Native | group="main" sCount=1 dsCount=0 obj=0x76384668 self=0xf0185900 | sysTid=24041 nice=-4 cgrp=default sched=0/0 handle=0xf337a534 | state=S schedstat=( 37276574736 7171909809 73050 ) utm=3082 stm=645 core=0 HZ=100 | stack=0xff730000-0xff732000 stackSize=8MB | held mutexes=
#00 pc 0000000000017758 /system/lib/libc.so (syscall+28)
#01 pc 00000000000b6605 /system/lib/libart.so (_ZN3art17ConditionVariable16WaitHoldingLocksEPNS_6ThreadE+92)
#02 pc 00000000003f1b17 /system/lib/libart.so (_ZN3artL12GoToRunnableEPNS_6ThreadE+230)
#03 pc 00000000003f1a07 /system/lib/libart.so (_ZN3art12JniMethodEndEjPNS_6ThreadE+8)
#04 pc 00000000006cdabd /system/framework/arm/boot-framework.oat (Java_android_os_BinderProxy_transactNative__ILandroid_os_Parcel_2Landroid_os_Parcel_2I+152)
at android.os.BinderProxy.transactNative (Native method)
at android.os.BinderProxy.transact (Binder.java:628)
at android.media.IAudioService$Stub$Proxy.getStreamVolume (IAudioService.java:1211)
at android.media.AudioManager.getStreamVolume (AudioManager.java:1295)
at com.google.android.gms.ads.internal.util.af.a (com.google.android.gms.dynamite_adsdynamite@17122050@17.1.22 (040406-245988633):11)
at com.google.android.gms.ads.nonagon.ad.activeview.c.a (com.google.android.gms.dynamite_adsdynamite@17122050@17.1.22 (040406-245988633):36)
at com.google.android.gms.ads.nonagon.ad.activeview.e.a (com.google.android.gms.dynamite_adsdynamite@17122050@17.1.22 (040406-245988633):18)
- locked <0x034e0bee> (a com.google.android.gms.ads.nonagon.ad.activeview.e)
at com.google.android.gms.ads.nonagon.ad.activeview.a.a (com.google.android.gms.dynamite_adsdynamite@17122050@17.1.22 (040406-245988633):6)
at com.google.android.gms.ads.internal.webview.k.a (com.google.android.gms.dynamite_adsdynamite@17122050@17.1.22 (040406-245988633):291)
at com.google.android.gms.ads.internal.webview.ae.a (unavailable:3)
at com.google.android.gms.ads.internal.webview.ad.run (unavailable:1)
at android.os.Handler.handleCallback (Handler.java:751)
at android.os.Handler.dispatchMessage (Handler.java:95)
at vq.a (com.google.android.gms.dynamite_adsdynamite@17122050@17.1.22 (040406-245988633):5)
at com.google.android.gms.ads.internal.util.f.a (com.google.android.gms.dynamite_adsdynamite@17122050@17.1.22 (040406-245988633):8)
at vq.dispatchMessage (com.google.android.gms.dynamite_adsdynamite@17122050@17.1.22 (040406-245988633):4)
at android.os.Looper.loop (Looper.java:154)
at android.app.ActivityThread.main (ActivityThread.java:6780)
at java.lang.reflect.Method.invoke! (Native method)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:1500)
at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1390)
```

**Comment by stowy** [comment]
This is a known internal bug, we have a fix which will be rolled out, date is pending.

**Comment by shewenbiao** [comment]
@stowy 
Thanks！ I am looking forward to the new version.

**Comment by ericleich** [comment]
Please use 17.2.1 - https://developers.google.com/admob/android/rel-notes#17.2.1

Hopefully crashes have already decreased since 8 days ago since we rolled out a play services APK release that should have had the fix, but 17.2.1 has the fix for the cases where the device's play services APK hasn't updated.

**Comment by shewenbiao** [comment]
@ericleich Thanks, I will try it.
